package.json

```diff
@@ -27,6 +27,7 @@
     "!**/.*"
   ],
   "scripts": {
+    "prepare": "make test-data",
```

Comment:
`yarn prepare` seems to automatically run when you do `yarn install`, although I don't see anything about it in the docs: https://yarnpkg.com/cli/install